src/datasource/components/QueryEditor.tsx

```diff
@@ -263,12 +263,20 @@ export class QueryEditor extends PureComponent<Props, State> {
   onPropertiesSelected = (sel: Array<SelectableValue<string>>) => {
     const { onChange, query, onRunQuery } = this.props;
     let properties: string[] = [];
+    let propertyDisplayNames: { [key: string]: string } = {};
```

Comment:
@Digized -- is this the part you expect to do the migration?  I think this will only work when in edit mode, but not from a read only saved dashbaord

Reply:
i get it now :) i added a suggestion for comment in the golang side